vochain/scrutinizer/process.go

```diff
@@ -138,7 +138,8 @@ func (s *Scrutinizer) ProcessCount(entityID []byte) int64 {
 		// If no entity ID, return the cached count of all processes
 		return atomic.LoadInt64(s.countTotalProcesses)
 	}
-	if c, err = s.db.Count(&indexertypes.Process{}, badgerhold.Where("EntityID").Eq(entityID)); err != nil {
+	if c, err = s.db.Count(&indexertypes.Process{},
+		badgerhold.Where("EntityID").Eq(entityID)); err != nil {
```

Comment:
I think there is an index missing here

Reply:
Please, review all indexes are used properly, that makes a big difference on the query cost